INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //body//div[@class='login-wrapper login-wrapper-modal']//div[@class='login-wrapper login-wrapper-modal']//div[1]//div[2] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:103 Cannot click on the element with locator: //body//div[@class='login-wrapper login-wrapper-modal']//div[@class='login-wrapper login-wrapper-modal']//div[1]//div[2] locatorType: xpath
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //form[@id='datashop-login-form']//input[@name='email'] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:119 Cannot send data on the element with locator: //form[@id='datashop-login-form']//input[@name='email'] locatorType: xpath
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //form[@id='datashop-login-form']//input[@name='password'] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:119 Cannot send data on the element with locator: //form[@id='datashop-login-form']//input[@name='password'] locatorType: xpath
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //form[@id='datashop-login-form']//button[contains(text(),'Sign in to continue')] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:103 Cannot click on the element with locator: //form[@id='datashop-login-form']//button[contains(text(),'Sign in to continue')] locatorType: xpath
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //div[@class='view-container']//p[@class='email'] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:161 Element not present with locator: //div[@class='view-container']//p[@class='email'] locatorType: xpath
INFO     Util:util.py:83 Actual Text From Application Web UI --> :: Innovaccer
INFO     Util:util.py:84 Expected Text From Application Web UI --> :: Innovaccer
INFO     Util:util.py:86 ### VERIFICATION CONTAINS !!!
INFO     TestStatus:teststatus.py:21 ### VERIFICATION SUCCESSFUL :: + Title Verified
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //div[@class='view-container']//p[@class='email'] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:161 Element not present with locator: //div[@class='view-container']//p[@class='email'] locatorType: xpath
ERROR    TestStatus:teststatus.py:24 ### VERIFICATION FAILED :: + Login was successful
INFO     TestStatus:selenium_driver.py:33 Screenshot save to directory: /Users/it000621/PycharmProjects/MB/base/../screenshots/Login was successful.1595236776189.png
ERROR    TestStatus:teststatus.py:43 test_validLoginTEST FAILED
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //body//div[@class='login-wrapper login-wrapper-modal']//div[@class='login-wrapper login-wrapper-modal']//div[1]//div[2] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:103 Cannot click on the element with locator: //body//div[@class='login-wrapper login-wrapper-modal']//div[@class='login-wrapper login-wrapper-modal']//div[1]//div[2] locatorType: xpath
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //form[@id='datashop-login-form']//input[@name='email'] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:119 Cannot send data on the element with locator: //form[@id='datashop-login-form']//input[@name='email'] locatorType: xpath
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //form[@id='datashop-login-form']//input[@name='password'] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:119 Cannot send data on the element with locator: //form[@id='datashop-login-form']//input[@name='password'] locatorType: xpath
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //form[@id='datashop-login-form']//button[contains(text(),'Sign in to continue')] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:103 Cannot click on the element with locator: //form[@id='datashop-login-form']//button[contains(text(),'Sign in to continue')] locatorType: xpath
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //div[@class='view-container']//p[@class='email'] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:161 Element not present with locator: //div[@class='view-container']//p[@class='email'] locatorType: xpath
INFO     Util:util.py:83 Actual Text From Application Web UI --> :: Innovaccer
INFO     Util:util.py:84 Expected Text From Application Web UI --> :: Innovaccer
INFO     Util:util.py:86 ### VERIFICATION CONTAINS !!!
INFO     TestStatus:teststatus.py:21 ### VERIFICATION SUCCESSFUL :: + Title Verified
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //div[@class='view-container']//p[@class='email'] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:161 Element not present with locator: //div[@class='view-container']//p[@class='email'] locatorType: xpath
ERROR    TestStatus:teststatus.py:24 ### VERIFICATION FAILED :: + Login was successful
INFO     TestStatus:selenium_driver.py:33 Screenshot save to directory: /Users/it000621/PycharmProjects/MB/base/../screenshots/Login was successful.1595236808109.png
ERROR    TestStatus:teststatus.py:43 test_validLoginTEST FAILED
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //body//div[@class='login-wrapper login-wrapper-modal']//div[@class='login-wrapper login-wrapper-modal']//div[1]//div[2] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:103 Cannot click on the element with locator: //body//div[@class='login-wrapper login-wrapper-modal']//div[@class='login-wrapper login-wrapper-modal']//div[1]//div[2] locatorType: xpath
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //form[@id='datashop-login-form']//input[@name='email'] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:119 Cannot send data on the element with locator: //form[@id='datashop-login-form']//input[@name='email'] locatorType: xpath
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //form[@id='datashop-login-form']//input[@name='password'] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:119 Cannot send data on the element with locator: //form[@id='datashop-login-form']//input[@name='password'] locatorType: xpath
WARNING  urllib3.connectionpool:connectionpool.py:665 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106095d68>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/5e660455bfe9c4df9ba53f52a6b54c7e/element
WARNING  urllib3.connectionpool:connectionpool.py:665 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106095dd8>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/5e660455bfe9c4df9ba53f52a6b54c7e/element
WARNING  urllib3.connectionpool:connectionpool.py:665 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106095cc0>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/5e660455bfe9c4df9ba53f52a6b54c7e/element
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //form[@id='datashop-login-form']//button[contains(text(),'Sign in to continue')] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:103 Cannot click on the element with locator: //form[@id='datashop-login-form']//button[contains(text(),'Sign in to continue')] locatorType: xpath
WARNING  urllib3.connectionpool:connectionpool.py:665 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1060959b0>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/5e660455bfe9c4df9ba53f52a6b54c7e/element
WARNING  urllib3.connectionpool:connectionpool.py:665 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1060ba390>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/5e660455bfe9c4df9ba53f52a6b54c7e/element
WARNING  urllib3.connectionpool:connectionpool.py:665 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1060ba160>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/5e660455bfe9c4df9ba53f52a6b54c7e/element
INFO     LoginPage:selenium_driver.py:70 Element not found with locator: //div[@class='view-container']//p[@class='email'] and  locatorType: xpath
INFO     LoginPage:selenium_driver.py:161 Element not present with locator: //div[@class='view-container']//p[@class='email'] locatorType: xpath
WARNING  urllib3.connectionpool:connectionpool.py:665 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1060ba518>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/5e660455bfe9c4df9ba53f52a6b54c7e/screenshot
WARNING  urllib3.connectionpool:connectionpool.py:665 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1060ba6d8>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/5e660455bfe9c4df9ba53f52a6b54c7e/screenshot
WARNING  urllib3.connectionpool:connectionpool.py:665 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x106095160>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/5e660455bfe9c4df9ba53f52a6b54c7e/screenshot